bin/harvester-sb-collector.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -x
+
+HOST_PATH=${HARVESTER_HOST_PATH:-/}
+OUTPUT_DIR=${HARVESTER_CACHE_PATH:-/tmp/harvester-support-bundle}
+
+if [ -z "$HARVESTER_SUPPORT_BUNDLE_MANAGER_URL" ]; then 
+    echo "Environment variable HARVESTER_SUPPORT_BUNDLE_MANAGER_URL is not defined"
+    exit 1
+fi
+
+[ ! -e ${OUTPUT_DIR} ] && mkdir -p $OUTPUT_DIR
+
+NODE_NAME=${HARVESTER_NODE_NAME:-$(cat ${HOST_PATH}/etc/hostname)}
+BUNDLE_DIR="${OUTPUT_DIR}/${NODE_NAME}"
+
+mkdir -p ${BUNDLE_DIR}
+cd ${BUNDLE_DIR}
+# get some host information
+cp ${HOST_PATH}/etc/hostname .
+
+# collect logs
+mkdir -p logs
+cd logs
+dmesg &> dmesg.log
+
+# k3s logs don't rorate well and can be huge
+tail -c 100m ${HOST_PATH}/var/log/k3s-service.log > k3s-service.log
```

Comment:
Can we change it to 10m? 100m*N(nodes) can exceed github limit so community users cannot upload it on github in most cases.

Reply:
[Reduced to 10m.](b7d3b4f)